src/tests/system/tests/test_passkey.py

```diff
@@ -420,3 +420,81 @@ def test_passkey__su_same_key_for_multi_user(
             ioctl=f"{moduledatadir}/umockdev.ioctl",
             script=f"{testdatadir}/umockdev.script.{suffix}.{user}",
         )
+
+
+@pytest.mark.importance("high")
+@pytest.mark.ticket(jira="SSSD-7011", gh=7066)
+@pytest.mark.topology(KnownTopologyGroup.AnyAD)
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.builtwith(client="passkey", provider="passkey")
+def test_passkey__ssh_key_as_passkey_mapping_token(
```

Comment:
I think the function names and test titles might be confusing later on when someone is looking at this.   I think that was also @ikerexxe 's other comment earlier.

Maybe something like:

```
def test_passkey__check_mapping_with_ssh_key_is_invalid()
    :title: Check passkey mapping with invalid ssh key with AD, Samba, and LDAP server.
```

Reply:
updated.